doc/api.rst

```diff
@@ -357,6 +362,13 @@ Bytecode
 
       It is based on :meth:`to_concrete_bytecode` and so resolve jump targets.
 
+   .. method:: compute_stacksize() -> int
+
+      Compute the stacksize needed to execute the code.
+
+      This computation requires to build the control flow graph associated with
+      the code.
```

Comment:
Can you please mention that an exception (no need to mention which one) is raised if the bytecode is invalid.